packages/@aws-cdk/aws-eks/lib/shared-interfaces.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * EKS cluster control plane logging configuration
+ */
+export interface ControlPlaneLogging {
```

Comment:
Can we move this type to cluster.ts?

Reply:
i guess not, because we are using in both cluster.ts and cluster-resource.ts

Reply:
Okay so let’s rename this file to `logging.ts` please